2024-01-26 14:47:46,772 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:41]
2024-01-29 13:45:16,044 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:41]
2024-01-29 13:54:39,504 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:41]
2024-01-29 14:05:56,947 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:41]
2024-01-29 14:07:30,982 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:41]
2024-01-29 14:19:38,723 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:41]
2024-01-30 11:53:09,747 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 12:10:14,208 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:21:46,143 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:50:55,468 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:56:42,682 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:56:42,682 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:56:56,894 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:56:56,894 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:56:56,894 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:57:22,583 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:57:22,583 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:57:22,583 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:57:22,583 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:58:04,333 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:58:04,333 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:58:04,333 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:58:04,333 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 13:58:04,333 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 14:00:07,118 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 14:07:51,232 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 15:13:16,989 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 15:15:40,353 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 15:21:22,988 ERROR: Exception on /reset_password_request [POST] [in /home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py:825]
Traceback (most recent call last):
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/bluii/Documents/PyProject/microblog/app/routes.py", line 165, in reset_password_request
    send_password_reset_email(user)
  File "/home/bluii/Documents/PyProject/microblog/app/email.py", line 15, in send_password_reset_email
    text_body=render_template('email/reset_password.txt',
NameError: name 'render_template' is not defined
2024-01-30 15:22:56,961 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
2024-01-30 15:23:05,623 ERROR: Exception on /reset_password_request [POST] [in /home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py:825]
Traceback (most recent call last):
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask_mail.py", line 492, in send
    message.send(connection)
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask_mail.py", line 427, in send
    connection.send(self)
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask_mail.py", line 188, in send
    self.host.sendmail(sanitize_address(envelope_from or message.sender),
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 872, in sendmail
    self.ehlo_or_helo_if_needed()
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 611, in ehlo_or_helo_if_needed
    if not (200 <= self.ehlo()[0] <= 299):
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 451, in ehlo
    self.putcmd(self.ehlo_msg, name or self.local_hostname)
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 378, in putcmd
    self.send(f'{s}{CRLF}')
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 365, in send
    raise SMTPServerDisconnected('please run connect() first')
smtplib.SMTPServerDisconnected: please run connect() first

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/bluii/Documents/PyProject/microblog/app/routes.py", line 165, in reset_password_request
    send_password_reset_email(user)
  File "/home/bluii/Documents/PyProject/microblog/app/email.py", line 13, in send_password_reset_email
    send_mail('[Microblog] Reset Your Password',
  File "/home/bluii/Documents/PyProject/microblog/app/email.py", line 9, in send_mail
    mail.send(msg)
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask_mail.py", line 491, in send
    with self.connect() as connection:
  File "/home/bluii/Documents/PyProject/microblog/myenv/lib/python3.10/site-packages/flask_mail.py", line 152, in __exit__
    self.host.quit()
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 1004, in quit
    res = self.docmd("quit")
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 431, in docmd
    self.putcmd(cmd, args)
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 378, in putcmd
    self.send(f'{s}{CRLF}')
  File "/home/bluii/.pyenv/versions/3.10.13/lib/python3.10/smtplib.py", line 365, in send
    raise SMTPServerDisconnected('please run connect() first')
smtplib.SMTPServerDisconnected: please run connect() first
2024-01-30 15:28:04,184 INFO: Microblog startup [in /home/bluii/Documents/PyProject/microblog/app/__init__.py:43]
